CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ New features:
 
 Bugfixes:
 
+- GitHub `@actions/*` dependencies upgraded for post-October 2020 compatibility @jisantuc
```

Comment:
Do you mind also including the PR number? ie. `(#13 by @jisantuc)`. Also, I think it's worth noting that the `addPath` function is affected, and that we're upgrading because of security vulnerabilities as described in [this GitHub announcement](https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands)

Reply:
Sure -- 93a58d7